chain/neutrino.go

```diff
@@ -400,7 +399,7 @@ func (s *NeutrinoClient) Rescan(startHash *chainhash.Hash, addrs []ltcutil.Addre
 	}
 
 	s.clientMtx.Lock()
-/* commented out until ltcsuite/neutrino is updated
+	/* commented out until ltcsuite/neutrino is updated
```

Comment:
this section was commented out last time we updated, due to un-solved dependency issues.  We should un-comment it out now, to make it match btcsuite.

Reply:
I think PR #5 fixes this. Can you review that please?

Reply:
done  :+1: